src/lib/db/client-metrics-store-v2.ts

```diff
@@ -226,6 +259,41 @@ export class ClientMetricsStoreV2 implements IClientMetricsStoreV2 {
         return Object.values(tokens);
     }
 
+    async getMetricsForFeatureToggleV2(
```

Comment:
this is the same as the old code but parametrized with the hourly or daily table